db-drda/src/main/java/com/splicemachine/db/impl/drda/CodePoint.java

```diff
@@ -875,8 +877,16 @@
 	
 	// XID count
 	static final int XIDCNT = 0x1906;
-	
-	
+
+
+	// Security Plug-in
+	static final int PLGINLSE = 0x1910;
+	static final int PLGINLST = 0x191E;
+	static final int PLGINNM = 0x191C;
+	static final int PLGINCNT = 0x190F;
```

Comment:
please describe a bit what these are doing so we don't have to scan the 772-page DRDA docu.
Either here or at another central place.
e.g.

```
// PLGINCNT: plugin count (int32 count)
static final int PLGINCNT = 0x190F;

// PLGINNM: plugin name (string name)
static final int PLGINNM = 0x191C;

/// PLGINLST: plugin list (PLGINCNT, count x (PLGINLSE) )
static final int PLGINLST = 0x191E;

// PLGINLSE: plugin list entry (PLGINNM, [PLGINPPL])
static final int PLGINLSE = 0x1910;
```

Reply:
added, to net/CodePoint.java as well